### PR TITLE
Feat: create the sign in component

### LINKED ISSUE #29: Feat: create the sign in component
No need for any functionality, just the structure, and the correct markup. It will be injected inside the layout component #27

image

### PR DESCRIPTION
**Pull request by joliehobloss**
Fixes #29

### CODE REVIEW

**Review by nabigha-mogharbel — COMMENTED** [review]
What you did is good so far. I have like small note, if you can do it, it would be great and consistent with the Sign Up form i did. the idea is if you can put max-width to the whole SignIn div by 600 px so the input wont expand like this in desktop mode.

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,10 @@
...
-        <a
-          className="text-pink"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <SignIn />
```
Comment (wisammechano) [inline src/App.js]: please revert app.js and don't commit after testing.

File: src/components/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import signInLogo from '../assets/signInLogo.svg';
+
+const SignIn = () => {
+  return (
+    <div class="bg-transparent">
+      <div class="mt-12 ">
+        <img
+          src={signInLogo}
+          alt="images"
+          style={{ display: 'block', margin: '0 auto' }}
```
Comment (wisammechano) [inline src/components/SignIn.js]: `className="block mx-auto"` use tailwind instead

File: src/components/SignIn.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import signInLogo from '../assets/signInLogo.svg';
+
+const SignIn = () => {
+  return (
+    <div class="bg-transparent">
```
Comment (wisammechano) [inline src/components/SignIn.js]: wrap the whole thing by a wrapper using the container class

https://tailwindcss.com/docs/container

**Review by wisammechano — CHANGES_REQUESTED** [review]

**Review by wisammechano — APPROVED** [review]

**Pull request merged**